easybuild/easyconfigs/i/itac/itac-2019.4.036.eb

```
@@ -0,0 +1,21 @@
+name = 'itac'
+version = '2019.4.036'
+
+homepage = 'https://software.intel.com/en-us/intel-trace-analyzer/'
+description = """The Intel Trace Collector is a low-overhead tracing library that performs
+ event-based tracing in applications. The Intel Trace Analyzer provides a convenient way to monitor application
+ activities gathered by the Intel Trace Collector through graphical displays. """
+
+toolchain = SYSTEM
+
+sources = ['l_itac_p_%(version)s.tgz']
+checksums = ['8994f20c12ea314760482314677d278628cbe8bd79625b302b4d02c8ae69ca53']
+
+dontcreateinstalldir = 'True'
```

@ekieffer `dontcreateinstalldir` should be boolean:
```suggestion
dontcreateinstalldir = True
```